include/jemalloc/internal/tcache_inlines.h

```diff
@@ -32,13 +32,65 @@ tcache_enabled_set(tsd_t *tsd, bool enabled) {
 	tsd_slow_update(tsd);
 }
 
+static inline unsigned
+thread_nhbins_get(tsd_t *tsd) {
+	return tsd_thread_nhbins_get(tsd);
+}
+
+static inline void
+thread_nhbins_set(tsd_t * tsd, unsigned thread_nhbins) {
```

Comment:
thread_nhbins_set and thread_tcache_max_set have to be called together, right? If so let's enforce it, and not provide the separate setters. Because otherwise the setting order would matter too, e.g. for the last assert in this function.

Also consider only providing a combined getter function (but allow user to pass in NULL ptrs to only load one of the them).